.github/workflows/Tests.yaml

```diff
@@ -723,6 +723,13 @@ ${{ matrix.build_type }}-pch-${{ matrix.use_pch || 'ON' }}"
         # We currently don't require codecov in our guidelines, so don't fail
         # the CI build if codecov fails to upload
         continue-on-error: true
+      # Avoid running out of disk space by cleaning up the build directory
+      - name: Clean up unit tests
+        working-directory: build
+        run: |
+          ls | xargs du -sh
+          du -sh .
+          rm bin/Test_*
```

Comment:
```
rm -f bin/Test_*
```

Maybe also add a `pwd` so it's easy to know which directory we are seeing (I know it's build but I'll forget next week)